script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -436,7 +436,7 @@ As such, I stole it. The Knight defeated the Queen, claiming the throne for hims
 -- Page 546, Offset 10064.
 -- 「……強気だな。空想具現化も出来ぬほど弱っていながら、おれと正面から打ち合うと？」
 "Still standing firm?
- Though you have weakened so much that you cannot even deploy your Marble Phantasm... you still intend to face me head-on?"
+ Though you have weakened so much that you cannot even deploy your <Marble Phantasm|F a n t a s y   M a n i f e s t a t i o n        >... you still intend to face me head-on?"
```

Comment:
It should be the opposite

Fantasy Manifestation on the bottom, and Marble Phantasm on the top.